src/components/Treeview/TreeviewItem.Props.ts

```diff
@@ -22,17 +24,18 @@ export interface ITreeviewItem {
     checked?: boolean;
     parentId?: string;
     hoverOverBtn?: Array<IHoverOverBtn>;
+    className?: string;
 }
-export function MapChildren(item: ITreeviewItem, items: ITreeviewItem[]): ITreeviewItem[] {
 
-    let children = items.filter((element) => {
-        return (element.parentId === item.id);
+export function MapChildren(currentItem: ITreeviewItem, allItems: ITreeviewItem[]): ITreeviewItem[] {
```

Comment:
Maybe move this function to Treeview.tsx since it's only used there
And rename to SetChildrenForElement?